src/test/java/com/spotify/docker/it/PushPullIT.java

```diff
@@ -138,7 +138,7 @@ public void testPushImageToPrivateAuthedRegistryWithoutAuth() throws Exception {
     client.build(Paths.get(dockerDirectory), LOCAL_IMAGE);
 
     exception.expect(ImagePushFailedException.class);
-    exception.expectMessage("no basic auth credentials");
+    exception.expectMessage("Authentication is required");
```

Comment:
The registry seems to have been updated so this string changed: https://hub.docker.com/r/library/registry/tags/. If it changes again, I'll look into fixing the version we pull.